keygen/common/acss/acss_test.go

```diff
@@ -189,15 +191,113 @@ func TestSharedKey(t *testing.T) {
 	}
 }
 
-func TestPredicate(t *testing.T) {
+// Testcase predicate returns true
+func TestPredicateVerified(t *testing.T) {
+
 	curve := curves.K256()
-	k256 := curve.Point.Generator().(*curves.PointK256)
-	dealerPoint := k256.Generator()
-	var scalar = new(curves.ScalarK256).New(8)
-	key := SharedKey(scalar, dealerPoint)
-	commit := dealerPoint.ToAffineCompressed()
-	_, _, b := Predicate(key[:], []byte("foobarfoobarfoobarfoobarfoobarfoobar"), commit, 1, curve)
-	t.Log(b)
+	// KEYS
+	myKeyPair := GenerateKeyPair(curve)
+	dealerKeyPair := GenerateKeyPair(curve)
+
+	sharedKey := SharedKey(myKeyPair.PrivateKey, dealerKeyPair.PublicKey)
```

Comment:
Can you pull the latest `dev` code to your branch for this?
The encrypt and decrypt now uses the ecies so this shared key shudnt need to be created.

@ewynx